--- plugins_good/ext/jpeg/meson.build	2022-02-03 20:53:32.000000000 +0100
+++ plugins_good/ext/jpeg/meson.build	2022-03-09 10:33:53.947810300 +0100
@@ -5,8 +5,12 @@
   'gstjpegdec.c',
 ]
 
-jpeglib = dependency('libjpeg', required : get_option('jpeg'),
+jpeglib = dependency('libjpeg', required : false,
   fallback: ['libjpeg-turbo', 'jpeg_dep'])
+if not jpeglib.found()
+  c_comp = meson.get_compiler('c')
+  jpeglib = c_comp.find_library('libjpeg', has_headers: ['jpeglib.h'], required : get_option('jpeg'))
+endif
 
 if jpeglib.found()
   gstjpeg = library('gstjpeg',
